scripts/etc/init.d/rcS

```diff
@@ -760,7 +760,7 @@ cd /rootfs/boot/config/apt/
 # iterate through all the *.list files and add them to /etc/apt/sources.list.d
 for listfile in ./*.list
```

Comment:
this is repeatable using both ```/bin/sh``` and ```/bin/bash``` (under Ubuntu 14.04.02 LTS)

to actually get the desired functionality (as described in the reasoning for using ```./*.list```), you'd have to use the following:

```for file in $(ls *.bla 2> /dev/null); do echo ${file}; done```